### DISCUSSION TITLE
Is it planned to add periods to display fully filled period on the timeline?

### DISCUSSION THREAD

**Discussion by rrrrs09** [mark-when/parser#4] [Ideas]
It would be nice to add new token e.g. period to the parser for displaying fully filled area on the timeline like on the picture.
It would not be token like section or group that wraps amount of events.
That could be separate token like an event but outside of other tokens:
`12/2024 period: Name of period` or something similar

What do you think about this?

![image](https://github.com/user-attachments/assets/76d61666-f9af-47c1-9d51-0c4aacf91a21)

**Comment by kochrt** [comment]
If you tag an event with `#era` or `#milestone` it has that effect.

**Reply by rrrrs09** [reply]
Thank you so much! 👏 
But I can't find this in the docs. Is there page with this information?
